templates/DatasetMonitoring/projects.html.twig

```diff
@@ -35,17 +35,21 @@
                 <tr width="100%">
                     <td>
                         <div class="dotchart_legend" style="position: relative;">
-                            <span id="a"><strong>Dataset with status:</strong>
+                            <span><strong>Dataset with status:</strong>
                                 <img src="{{ asset('build/images/green-circle.png') }}"> Available:
+                                <span id="a"></span>
```

Comment:
Should we rename the id's

Reply:
I have to agree. 'Accepted', 'Registered', 'Identified' was some of the old nomenclature.